src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala

```diff
@@ -288,7 +203,12 @@ class FeelInterpreter {
       // functions
       case FunctionInvocation(name, params) =>
         withFunction(findFunction(context, name, params),
-                     f => invokeFunction(f, params))
+          f => invokeFunction(f, params) match {
+            case ValError(failure) =>
+              error(EvaluationFailureType.FUNCTION_INVOCATION_FAILURE, s"Failed to invoke function '$name': $failure")
+              ValNull
```

Comment:
❓ Isn't this a behavioural change? I get that we're in the process of changing errors to nulls. I just didn't expect it to be part of this PR.

Reply:
You're right. :+1: 

I could remove it from the PR. However, since we are planning to do this change anyway, I will just leave it here. :shushing_face: